framework/helpers/BaseStringHelper.php

```diff
@@ -457,17 +457,23 @@ public static function mb_ucfirst($string, $encoding = 'UTF-8')
      * @param string $string the string to be proceeded
      * @param string $encoding Optional, defaults to "UTF-8"
      * @return string
-     * @see https://secure.php.net/manual/en/function.ucwords.php
+     * @see https://www.php.net/manual/en/function.ucwords
      * @since 2.0.16
      */
     public static function mb_ucwords($string, $encoding = 'UTF-8')
     {
-        $words = preg_split("/\s/u", $string, -1, PREG_SPLIT_NO_EMPTY);
+        if (empty($string)) {
```

Comment:
```suggestion
        if ($string === '') {
```

else it would convert `'0'` to `''`.

Reply:
@samdark fixed